man/systemd-boot.xml

```diff
@@ -257,12 +257,13 @@
 
       <varlistentry>
         <term><keycap>Esc</keycap></term>
+        <term><keycombo><keycap>Ctrl</keycap><keycap>c</keycap></keycombo></term>
         <listitem><para>Abort the edit and quit the editor</para></listitem>
       </varlistentry>
 
       <varlistentry>
         <term><keycombo><keycap>Ctrl</keycap><keycap>k</keycap></keycombo></term>
-        <listitem><para>Clear the command line</para></listitem>
+        <listitem><para>Clear the command line forwards</para></listitem>
```

Comment:
why drop this? it only deletes to the end of line from the current location, no?

Reply:
I'm not dropping anything here, just clarifying. `Clear the command line` makes it sound like it deletes the whole thing, when it really just deletes forwards (just like your typical shell does).

Reply:
oops, sorry, I inverted the patch in my head. sorry for the confusoin